Makefile

```diff
@@ -14,3 +14,7 @@ coverage:
 lint:
 	# sudo apt-get install python-flake8
 	$(PYTHON) -m flake8 .
+
+.PHONY: install-dev
+install-dev:
+	ln -s $(shell pwd)/txjuju /usr/local/lib/python2.7/dist-packages/txjuju
```

Comment:
I wasn't going to worry about it since it's developer-focused and ln will loudly fail if the target already exists.  Silently failing means you won't know if the target is the symlink you wanted or something else (e.g. you'd installed an older version earlier).  So I'd rather leave it.